src/compressed_tensors/compressors/model_compressor.py

```diff
@@ -252,9 +252,17 @@ def decompress(self, model_path: str, model: Module):
             setattr(model, SPARSITY_CONFIG_NAME, self.sparsity_compressor.config)
 
         if self.quantization_compressor is not None:
-            apply_quantization_config(model, self.quantization_config)
+            names_to_scheme = apply_quantization_config(model, self.quantization_config)
             load_pretrained_quantization(model, model_path)
-            dense_gen = self.quantization_compressor.decompress(model_path)
+            if (
+                self.quantization_config.format
+                == CompressionFormat.pack_quantized.value
+            ):
+                dense_gen = self.quantization_compressor.decompress(
+                    model_path, names_to_scheme=names_to_scheme
+                )
+            else:
+                dense_gen = self.quantization_compressor.decompress(model_path)
```

Comment:
Instead of an if statement here could we just update the parent decompress method to take in kwargs so we can pass the names in regardless? This would make it easier when we want to add this info to future compressors. We do this for the compress method: `def compress(self, model_state: Dict[str, Tensor], **kwargs) -> Dict[str, Tensor]:` then any compressors that need the mapping are:

```
    def compress(
        self,
        model_state: Dict[str, Tensor],
        model_quant_args: Dict[str, QuantizationArgs],
        **kwargs,
    ) -> Dict[str, Tensor]:
```